### PR TITLE
chore(Issue Templates) Fix improve-existing-docs.yaml grammar issues

### PR DESCRIPTION
**Pull request by ArjunSharda**
### Summary

Fixes grammar issue possibly in improve-existing-docs.yaml

### Reason
Changes wording as per what I think should be right

review:copyedit is needed

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/improve-existing-docs.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ body:
       label: What did you expect to happen?
       description: Tell us what you think should have happened.
       placeholder: |
-          "This feature should have been documented with {X}, {Y}, and {Z} details."
+          "This feature should have been documented with details such as {X}, {Y}, and {Z}"
```
Comment (Guaris) [inline .github/ISSUE_TEMPLATE/improve-existing-docs.yaml:39]:
```suggestion
          "This feature should have been documented with the following details: {X}, {Y}, and {Z}"
```

How do you feel about this revision?

Comment (Guaris) [reply .github/ISSUE_TEMPLATE/improve-existing-docs.yaml:39]: Thanks for contributing. Let me know what you think about my suggestion.

Comment (ArjunSharda) [reply .github/ISSUE_TEMPLATE/improve-existing-docs.yaml:39]: It's nice! I will commit it.

File: .github/ISSUE_TEMPLATE/improve-existing-docs.yaml [hunk 2]
```diff
@@ -27,7 +27,7 @@ body:
       description: Tell us what the problem is.
       placeholder: |
           "I wanted information about this feature, but couldn't find it in the docs!"
-          "The instructions for {X} didn't work as documented."
+          "The documentation for {X} didn't work as documented."
```
Comment (Guaris) [inline .github/ISSUE_TEMPLATE/improve-existing-docs.yaml:30]:
```suggestion
          "The documentation for {X} didn't work as expected."
```

File: .github/ISSUE_TEMPLATE/improve-existing-docs.yaml [hunk 3]
```diff
@@ -16,7 +16,7 @@ body:
     id: link
     attributes:
       label: Where is the problem?
-      description: Link to the affected page(s)
+      description: Provide the link(s) to the affected page(s)
```
Comment (Guaris) [inline .github/ISSUE_TEMPLATE/improve-existing-docs.yaml:19]:
```suggestion
      description: Provide a link to the affected page(s).
```

**Review by Guaris — CHANGES_REQUESTED** [review]

**Comment by Guaris** [comment]
few more revisions and we can merge this

File: .github/ISSUE_TEMPLATE/improve-existing-docs.yaml (hunk 3 above)
Comment (ArjunSharda) [reply .github/ISSUE_TEMPLATE/improve-existing-docs.yaml:19]: I don't think this is right.

Comment (ArjunSharda) [reply .github/ISSUE_TEMPLATE/improve-existing-docs.yaml:19]: I will add your change though and change another word.

**Review by Guaris — CHANGES_REQUESTED** [review]

**Review by lena-larionova — APPROVED** [review]

**Pull request merged**